docs/cloud/runtime-troubleshooting-with-functions.md

```diff
@@ -1,34 +1,27 @@
-# Run-time troubleshooting with Functions
+# Top tab
 
-Netdata Functions feature allows you to execute on-demand a pre-defined routine on a node where a Netdata Agent is running. These routines are exposed by a given collector. 
-These routines can be used to retrieve additional information to help you troubleshoot or to trigger some action to happen on the node itself.
+The Top tab allows you to run [Netdata Functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md) on a node where a Netdata Agent is running. These routines are exposed by a given collector.
+They can be used to retrieve additional information to help you troubleshoot or to trigger some action to happen on the node itself.
 
+> **Tip**
+>
+> You can also execute a Function from the [Nodes tab](https://github.com/netdata/netdata/blob/master/docs/dashboard/nodes-tab.md), by pressing the `f(x)` button.
 
-### Prerequisites
+> **Note**
+>
+> If you get an error saying that your node can't execute Functions please check the [prerequisites](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md#prerequisites).
 
-The following is required to be able to run Functions from Netdata Cloud.
-* At least one of the nodes claimed to your Space should be on a Netdata agent version higher than `v1.37.1`
-* Ensure that the node has the collector that exposes the function you want enabled ([see current available functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md#what-functions-are-currently-available))
+The main view of this tab provides you with -depending on the Function- two elements: a visualization on the top and a table on the bottom.
```

Comment:
> -depending on the Function- two elements

Typos?

Reply:
maybe I need to encase it in parenthesis instead, we do this in Greek, but maybe looks odd in english